cogs/blacklist.py

```diff
@@ -2,16 +2,17 @@
 from discord import slash_command, option
 from discord.ext import commands
 from discord.ext.bridge import Bot
+
```

Comment:
Why?

Reply:
To seperate third party imports (discord) from local imports (utilities.database)